packages/scene-composer/src/augmentations/components/three-fiber/anchor/__tests__/AnchorWidget.spec.tsx

```diff
@@ -92,53 +99,48 @@ describe('AnchorWidget', () => {
 
   it('should render correctly', () => {
     setStore('test-ref', 'test-ref');
-    const container = renderer.create(<AnchorWidget node={node as any} defaultIcon={DefaultAnchorStatus.Info} />);
+    const container = renderer.create(<AnchorWidget node={node} defaultIcon={DefaultAnchorStatus.Info} />);
 
     expect(container).toMatchSnapshot();
   });
 
   it('should render correctly with non default tag settings', () => {
     setStore('test-ref', 'test-ref');
     (useTagSettings as jest.Mock).mockReturnValue({ scale: 3, autoRescale: true });
-    const container = renderer.create(<AnchorWidget node={node as any} defaultIcon={DefaultAnchorStatus.Info} />);
+    const container = renderer.create(<AnchorWidget node={node} defaultIcon={DefaultAnchorStatus.Info} />);
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render correctly when not visible', () => {
+    setStore('test-ref', 'test-ref', true, false);
+    const container = renderer.create(<AnchorWidget node={node} defaultIcon={DefaultAnchorStatus.Info} />);
 
     expect(container).toMatchSnapshot();
```

Comment:
This is really good feedback & appreciate the catch! However, the change here was to implement an improvement on use of `any`. Pre-existing logic wasn't changed otherwise. Would suggest not blocking on this as we should encourage these types of improvements.